M3T2/Tests/DatabaseFixture.cs

```diff
@@ -0,0 +1,19 @@
+﻿[SetUpFixture]
+public class DatabaseFixture
+{
+    [OneTimeSetUp]
+    public async Task OneTimeSetUp()
```

Comment:
This sounds like an easy-to-go solution, and may work for small projects, but, this would not be ok for larger ones. To be able to trust the tests they should be run very often and give instant feedback (in other cases they won't be used). No need to do anything, but I would suggest checking https://github.com/jbogard/Respawn  (https://www.youtube.com/watch?v=E4TeWBFzcCw)